src/electron/release_windows_action.sh

```diff
@@ -30,11 +30,13 @@ cat > build/env.nsh << EOF
 !define SENTRY_DSN "https://sentry.io/api/159502/store/?sentry_version=7&sentry_key=6a1e6e7371a64db59f5ba6c34a77d78c"
 EOF
 
+# Set-up electron to only download new updates but never upload new ones
```

Comment:
What do you mean with "electron to only download new updates but never upload new ones"?
This comment could be clarified.

Reply:
What I'm trying to express is that we aren't ever using electron to publish our binaries to s3, we're only using it so clients can download updates

Reply:
Oh I see. So the flags are only setting where the client should fetch updates from.
Is publishing the default behavior? Is there a flag that says "don't publish"?

Maybe this will be clearer:
```sh
electron-builder \
  --win \
  --publish never \
  --config src/electron/electron-builder.json \
  --config.extraMetadata.version=$(scripts/semantic_version.sh -p windows) \
  --config.win.certificateSubjectName='Jigsaw Operations LLC' \
  # Set-up where to fetch updates from
  --config.publish.provider=s3 \
  --config.publish.bucket=outline-releases \
  --config.publish.path=client
```

Reply:
`--publish never` would be that option ^_^

That said, comments were improved in https://github.com/Jigsaw-Code/outline-client/pull/618/commits/2f079d3e809399197e3ea0c3005c3db61e9e5b3d